webinterface/pages/10_Quant_LFQ_DIA_ion_ZenoTOF.py

```diff
@@ -102,15 +107,20 @@ def _main_page(self) -> None:
                 type="secondary",
                 help="link to the module documentation",
             )
-            if self.variables_dia_quant.alpha_warning:
-                st.warning(
-                    "This module is in ALPHA phase. It has not yet passed peer review and should be used with caution.",
-                    icon="🚨",
+            if self.variables_dia_quant.archived_warning:
+                st.info(
+                    self.variables_dia_quant.texts.ShortMessages.warning_archived
                 )
-            if self.variables_dia_quant.beta_warning:
-                st.warning(
-                    "This module is in BETA phase. The figure presented below and the metrics calculation may change in the near future."
+            else:
+                if self.variables_dia_quant.alpha_warning:
```

Comment:
this is repeated from above. I am not sure how you want the warnings exactly to display, but I think this can go into an function, checking ing if the configuration sets these flags and then it calls the display method.

Reply:
yes. I will quickly have a look.